src/main/java/frc/robot/RobotContainer.java

```diff
@@ -83,7 +83,7 @@ public class RobotContainer {
     private final Climber climber = new Climber();
     private final Vision vision = new Vision();
     private final IntakeSubsystem intake = new IntakeSubsystem();
-    private final PowerMonitor powerMonitor = new PowerMonitor();
+//    private final PowerMonitor powerMonitor = new PowerMonitor();
```

Comment:
Should we at least leave a comment why this is commented out? Or just remove it if we don't need it.